File: C/queue.c
#include <stdio.h>
#include <stdlib.h>
void init(int *head, int *tail){
    *head = *tail = 0;
}
void enQ(int *q,int *tail, int element){
    q[(*tail)++] = element;
}
int dQ(int *q,int *head){
    return q[(*head)++];
}
void printQ(int *q,int head,int tail){
    int i = tail - 1;
    while(i >= head)
        printf("%d ",q[i--]);
}
int main(){
    int head, tail;
    int queue[5];
    init(&head,&tail);
    enQ(queue,&tail, 3);
    enQ(queue,&tail, 4);
    enQ(queue,&tail, 7);
    //dQ(queue, &head);
    printQ(queue,head,tail);
    return 0;
}

File: Python/queue.py
class Queue:
    def __init__(self, n):
        self.l=[n]
    def enQ(self, n):
        #enqueue but short
        self.l.append(n)
    def dQ(self):
        #dequeque but shorthand
        temp=self.l[0]
        self.l=self.l[1:]
        return temp
    def peek(self):
        return self.l[0]
    def __str__(self):
        return str(self.l)
    def size(self):
        return len(self.l)
    def isEmpty(self):
        return self.size()==0
if __name__=='__main__':
    q=Queue(3)
    q.enQ(5)
    q.enQ(60)
    q.dQ()
    q.dQ()
    print(q.dQ())
    print(q.isEmpty())